activedata_etl/transforms/cov_to_es.py

```diff
@@ -59,21 +62,21 @@ def process(source_key, source, destination, resources, please_stop=None):
         for artifact in artifacts:
             try:
                 if "jscov" in artifact.name:
-                    pass
-                    # coverage_artifact_exists = True
-                    # _, artifact_etl = etl_header_gen.next(source_etl=parent_etl, url=artifact.url)
-                    # if DEBUG:
-                    #     Log.note("Processing jscov artifact: {{url}}", url=artifact.url)
-                    #
-                    # keys.extend(process_jscov_artifact(
-                    #     source_key,
-                    #     resources,
-                    #     destination,
-                    #     task_cluster_record,
-                    #     artifact,
-                    #     artifact_etl,
-                    #     please_stop
-                    # ))
+                    coverage_artifact_exists = True
+                    _, artifact_etl = etl_header_gen.next(source_etl=parent_etl, url=artifact.url)
+                    if DEBUG:
+                        Log.note("Processing jscov artifact: {{url}}", url=artifact.url)
+
+                    keys.extend(process_jscov_artifact(
+                        source_key,
+                        resources,
+                        destination,
+                        task_cluster_record,
+                        artifact,
+                        artifact_etl,
+                        DO_AGGR,
```

Comment:
Do not make this a parameter, put `DO_AGGR` into the `process_jscov_artifact` module

Reply:
Okay.